Examples/Geometry/AxesActor/index.js

```diff
@@ -0,0 +1,80 @@
+import '@kitware/vtk.js/favicon';
+
+// Load the rendering pieces we want to use (for both WebGL and WebGPU)
+import '@kitware/vtk.js/Rendering/Profiles/Geometry';
+
+import macro from '@kitware/vtk.js/macros';
+import vtkAxesActor from 'vtk.js/Sources/Rendering/Core/AxesActor';
+import vtkFullScreenRenderWindow from '@kitware/vtk.js/Rendering/Misc/FullScreenRenderWindow';
+
+import controlPanel from './controlPanel.html';
+
+console.warn(
+  'Click on index.ts to open source code for this example --------->'
+);
+
+// ----------------------------------------------------------------------------
+// Standard rendering code setup
+// ----------------------------------------------------------------------------
+
+const fullScreenRenderer = vtkFullScreenRenderWindow.newInstance({
+  background: [0.9, 0.9, 0.9],
```

Comment:
only 2 examples have this background color. Can you please choose another color that is more popular in the code base (0, 0.1, 0.3, 0.5 or 1).

(Eventually we should change the other 2 instances as well...)